woocommerce/woocommerce
# Adjust spacing in/between elements on the Cart and Checkout block pages

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate. 



### Changes proposed in this Pull Request:





Closes #42108.

With this PR, we're adjusting spacing in/between elements on the Cart and Checkout block pages as per the design, 


### How to test the changes in this Pull Request:


1. Add a normal product to the site (don't add subscription product). 
2. Visit the Cart and Checkout pages
3. Confirm both the Cart and Checkout pages have spacing as per the design attached below. 
4. Confirm separator in order summary is not visible. 
![image](https://github.com/woocommerce/woocommerce/assets/11503784/c7f4d533-85b3-4ede-8c3f-663ed8a53f3e)
5. Add subscription product to the site. 
6. Visit the cart page and confirm the subscription details are aligned with product name and product price. 
7. Confirm the borders are uniform in color. 
![image](https://github.com/woocommerce/woocommerce/assets/11503784/5e148dfe-229a-4226-8723-42792a34fbc8)
8. Visit Checkout page and confirm the contact form is aligned with order summary. 
<img width="1383" alt="image" src="https://github.com/woocommerce/woocommerce/assets/11503784/217274c2-7055-4a56-97e2-cda28d78875f">
9. Confirm the shipping are not tight have gaps as per the design: 
![image](https://github.com/woocommerce/woocommerce/assets/11503784/85b6c2f9-1168-4bd5-b2ed-018c809a5ad1)





### Design
Figma link: 2JyXNimI4oGeJZhgQNZE9V-fi-819_26489

<img width="1273" alt="image" src="https://github.com/woocommerce/woocommerce/assets/11503784/6f4e9a07-b0b6-4297-9503-3d144b72a0ea">

![image](https://github.com/woocommerce/woocommerce/assets/11503784/7d513c4c-8a2b-4e8a-b784-bd30e5ede294)
![image](https://github.com/woocommerce/woocommerce/assets/11503784/f84c36f8-b806-4e2f-a2d5-560bd6b50414)

![image](https://github.com/woocommerce/woocommerce/assets/11503784/aa1eee8e-f950-4fad-844c-f6c655116d2a)



### Changelog entry




<details>

#### Significance




#### Type




#### Message 
Adjust spacing in/between elements on the Cart and Checkout block pages.

</details>
